electricpy/capacitor.py

```diff
@@ -0,0 +1,316 @@
+################################################################################
```

Comment:
Were you sneaking a peek at the `develop` branch? :wink: 

Hah!

Anyway, it's about time we get this in here... Let's do it!

Reply:
In latest commit i have replicated the `dev` branch folder structure please review it and give your valuable feed back